.env.example

```diff
@@ -9,7 +10,7 @@ COMMUNITY_API_HOST="https://community-api-secure.test.delius.probation.hmpps.dsd
 COMPLEXITY_API_HOST="https://complexity-of-need-staging.hmpps.service.justice.gov.uk"
 ASSESSMENT_API_HOST="https://offender-dev.aks-dev-1.studio-hosting.service.justice.gov.uk"
 ASSESS_RISKS_AND_NEEDS_API_HOST="https://assess-risks-and-needs-dev.hmpps.service.justice.gov.uk"
-TIERING_API_HOST: https://hmpps-tier-dev.hmpps.service.justice.gov.uk
+TIERING_API_HOST="https://hmpps-tier-dev.hmpps.service.justice.gov.uk"
```

Comment:
Oooops. Good spot!

Reply:
To YAML or not to YAML, that is the question